src/eradiate/quad.py

```diff
@@ -57,6 +57,14 @@ def _nodes_weights_validator(self, attribute, value):
                 f"and weights.shape = {self.weights.shape}"
             )
 
+    def __hash__(self) -> int:
```

Comment:
I can't see why; this may goes back to a time when I wanted to have bins in a set for `BinSet` objects. Now these `BinSet` objects take a `list[Bin]` as parameter so that hash is useless. Removing it.